docs/starter/cli-usage.md

```diff
@@ -0,0 +1,83 @@
+# Using the PWABuilder CLI
+
+The PWABuilder allows you to create new progressive web app projects from a template in just a few moments. This article has more in-depth coverage on usage of the CLI. If you want to get started quickly with the PWA Starter, check out the [Quick Start.](/starter/quick-start)
+
+## create
+
+The `create` command allows you to create a new progressive web app project from one of our templates.
+
+#### Basic usage
+
+```
+pwa create
+```
+
+This command will prompt you for a name for your app fetch the default version of the PWA Starter template. The name you enter for your app will be used throughout the template.
+
+#### Pass a name as an argument
+
+If you want to skip prompting, you can pass the name argument directly to the command like so:
+
+```
+pwa create <app-name>
+```
+
+This will directly create your application directory without prompting for a username.
```

Comment:
IMO, I'd like to keep it simple and mostly focus on `pwa create myApp` with the name in the command as our default recommendation. I see the prompting as a fail safe, if they forget to include the name